sdk/communication/tests.yml

```diff
@@ -0,0 +1,59 @@
+trigger: none
+
+parameters:
+  - name: Services
+    type: object
+    default:
+      - azure-communication-chat
+      - azure-communication-email
+      - azure-communication-identity
+      - azure-communication-networktraversal
+      - azure-communication-phonenumbers
+      - azure-communication-rooms
+      - azure-communication-sms
+
+stages:
+  - ${{ each service in parameters.Services }}:
+    - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+      parameters:
+        JobName: ${{ replace(service, '-', '_') }}
+        BuildTargetingString: ${{ service }}
```

Comment:
```suggestion
```

Reply:
I suggest that you remove the BuildTargetingString from the parameters here so that you can contract the test footprint at queue time.

Reply:
Have commited the suggestion itself 👍